components/DesktopMenu/DesktopMenu.tsx

```diff
@@ -1,20 +1,44 @@
+import * as React from "react"
 import { ChevronUpIcon, ChevronDownIcon } from "@heroicons/react/24/solid"
 import Image from "next/image"
 import { useState } from "react"
 import CustomConnectWallet from "../CustomConnectWallet"
+import { ToggleButton } from "../../shared/Button"
+import { useTheme } from "../../providers/ThemeProvider"
 
 const DesktopMenu = () => {
+  const { onChangeThemeConfig, themeMode } = useTheme()
+
   const [isMenuOpen, setIsMenuOpen] = useState(false)
   const toggleMenu = () => setIsMenuOpen(!isMenuOpen)
+  const [toggle, setToggle] = useState(false)
```

Comment:
can you change this to `isDarkMode` to make it distinct from `isMenuOpen`?

current name: `toggle` seems like a function call while this is a `boolean` right?